src/definitions/elements/step.less

```diff
@@ -251,49 +381,49 @@
 
 /* Mobile (Default) */
 @media only screen and (max-width: (@largestMobileScreen)) {
-    .ui.steps:not(.unstackable) {
+    .ui.steps:not(.unstackable, .railing) {
```

Comment:
If you keep this css level 4 selector syntax, this has to wait until until 2.11 or even 3.0 as only browsers from January 2021 are supporting this syntax.
See https://github.com/fomantic/Fomantic-UI/discussions/2854 for voting results of new CSS level 4 feature inclusion poll

To avoid this use `:not(.unstackable):not(.circular)` instead

Reply:
Thanks for your clarification. Will update PR very soon.